### PR TITLE
Popup window

### PR DESCRIPTION
**Pull request by dazehinn**
Created a popup window for the details in the projects section.

When each of the 'See Project' buttons in the 'Portfolio' section is clicked, it opens a pop-up window showing the details of the project.

This was done by creating popup.js and popup.css files.

### CODE REVIEW

**Review by GedeonTS — CHANGES_REQUESTED** [review]
# INVALID REVIEW REQUEST

Hi @dazehinn , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has a Pull Request message that is not descriptive enough please add more details

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: popups.js [hunk 1]
```diff
@@ -0,0 +1,414 @@
...
+        background: '#ebebff',
+        borderRadius: '8px',
+        fontFamily: "'Poppins', sans-serif",
+        fontStyle: 'normal',
+        fontWeight: 500,
+        fontSize: '12px',
+        letterSpacing: '0.03em',
+        color: '#6070ff',
+        margin: '5px',
+      };
```
Comment (brainconnect93) [inline popups.js]: - [ ] I would kindly suggest you stick to the normal way of styling using the external CSS file for now for easy accessibility and navigation 👍 👍

File: index.html [hunk 2]
```diff
@@ -67,9 +70,9 @@ <h2 class="project-title">Tonic</h2>
...
                    <li>javaScript</li>
                </ul>
-               <button class="action-button">See Project</button>
+               <button class="action-button see-project">See Project</button>
              </div>
-           </div>
+            </div>
 
          <div class="card">
              <img src="./img/Snapshoot Portfolio (1).png" alt="snapshot poortfolio" class="snapshot">
```
Comment (brainconnect93) [inline index.html:79]: - [ ] Having created your cards dynamically using `JavaScript`, I would kindly suggest you remove the code for the cards you have in your `HTML` file, With this you have show the best practice in javascript and as well follow the DRY method of coding practices.

File: popup.css [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+}
+
+.popup-close-btn {
+  color: #67798e;
+  font-size: 24px;
+  font-weight: 900;
+  position: absolute;
+  top: 35px;
+  right: 32px;
+}
```
Comment (brainconnect93) [inline popup.css:28]: - [ ] I would kindly suggest we stick to the Figma design as much as possible, currently the `desktop` and `mobile` versions of your pop-up modal look quite different from what we have in the Figma template, the alignment, and the positioning.

Your design 👇 


| mobile screen  | desktop screen |
| ------------- | ------------- |
|  sam1 | sam  |


Figma design 👇 

| mobile screen | desktop screen |
| --- | --- |
| real-pop | pp1-real |


  - kindly check the text alignment, the image size, the background content blur, and the button for the desktop and mobile versions.

Thank you.

**Review by brainconnect93 — CHANGES_REQUESTED** [review]
# STATUS: Changes Required :recycle:

### Hi @dazehinn, 👋 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



### To Highlight :1st_place_medal: 
-  Has a proper Pull Request. ✔️  
- No Linter errors :heavy_check_mark: 
- Good use of Github flow ✔️ 

### Changes Required

- [x]  _Please check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @brainconnect93 ) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
Comment (dazehinn) [reply index.html:79]: @brainconnect93 
Thanks for the time taken to review my code.  I do have some concerns. When you say I should remove the code for the card in my HTML file, do you mean I delete it? If yes, does that mean I have to create both the cards and the popup windows dynamically?

**Review by mckent05 — APPROVED** [review]
Hi @dazehinn, 

# STATUS: APPROVED ✔️ ✔️ 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations and well done! 🎉 
approved


## Highlights
- All linter checks are passing ✔️ ✔️ 
- Your PR is professional 👏 👏 
-  Your popup window works as expected 👏 👏 

## Optional suggestions
**OPTIONAL** - Your third project card is not aligned as properly, please fix this 🙏 🙏. Remember it is important that you try to replicate the Figma design as closely as possible to give confidence to recruiters in your ability to produce exact business requirements 🙏
efefefe
**OPTIONAL** - Your popup window for the desktop version is not aligned properly, please fix this 🙏 🙏 .
ggjqwgbqwgb


_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**